core/tests/karma.conf.ts

```diff
@@ -115,7 +115,7 @@ module.exports = function(config) {
       terminal: argv.terminalEnabled
     },
     // Continue running in the background after running tests.
-    singleRun: true,
+    singleRun: false,
```

Comment:
Why this change? I think enabling `singleRun` lets us reuse the same browser for all the tests, which should speed up test runs. See https://karma-runner.github.io/6.4/config/configuration-file.html#singlerun

Reply:
Crept in by mistake!
Thanks, removed